app/faucet/views.py

```diff
@@ -104,7 +103,8 @@ def process_faucet_request(request, pk):
     except FaucetRequest.DoesNotExist:
         raise Http404
 
-    faucet_amount = settings.FAUCET_AMOUNT
+    """Converting gwei to eth by dividing the gwei amount by 1000000000"""
+    faucet_amount = settings.FAUCET_AMOUNT + (recommend_min_gas_price_to_confirm_in_time(5) / 1000000000)
```

Comment:
upon further review / experimentation in my console, i think that this should be 

> faucet_amount = settings.FAUCET_AMOUNT * recommend_min_gas_price_to_confirm_in_time(5)

since the base faucet amount assumes 1 gwei and the total gas the user will need to use will be `gasPrice` * `gasLimit`